openmetadata-service/src/main/java/org/openmetadata/service/clients/pipeline/airflow/AirflowRESTClient.java

```diff
@@ -292,7 +292,7 @@ public PipelineServiceClientResponse getServiceStatusInternal() {
       // APIs URL not found
       if (response.statusCode() == 404) {
         return buildUnhealthyStatus(
-            "Airflow APIs not found. Please follow the installation guide.");
+            "Airflow APIs not found. Please validate if the OpenMetadata Airflow plugin is installed correctly. Follow [this guide](https://docs.open-metadata.org/v1.2.x/deployment/ingestion/openmetadata) for further details.");
```

Comment:
```suggestion
            "Airflow APIs not found. Please validate if the OpenMetadata Airflow plugin is installed correctly. Follow [this guide](https://docs.open-metadata.org/deployment/ingestion/openmetadata) for further details.");
```

to pick the latest version